src/Microsoft.Health.Fhir.TemplateManagement/TemplateLayerParser.cs

```diff
@@ -17,21 +18,21 @@ namespace Microsoft.Health.Fhir.TemplateManagement
 {
     public static class TemplateLayerParser
     {
-        public static TemplateLayer ParseArtifactsLayerToTemplateLayer(ArtifactLayer artifactsLayer)
+        public static TemplateLayer ParseArtifactsLayerToTemplateLayer(OCIArtifactLayer artifactsLayer)
         {
             TemplateLayer oneTemplateLayer = new TemplateLayer();
             var artifacts = DecompressRawBytesContent((byte[])artifactsLayer.Content);
             var parsedTemplate = ParseToTemplates(artifacts);
 
-            oneTemplateLayer.Content = parsedTemplate;
+            oneTemplateLayer.TemplateContent = parsedTemplate;
             oneTemplateLayer.Digest = artifactsLayer.Digest;
             oneTemplateLayer.Size = artifacts.Sum(x => x.Value == null ? 0 : x.Value.Length);
             return oneTemplateLayer;
         }
 
-        public static Dictionary<string, string> DecompressRawBytesContent(byte[] rawBytes)
+        public static Dictionary<string, byte[]> DecompressRawBytesContent(byte[] rawBytes)
```

Comment:
>DecompressRawBytesContent [](start = 49, length = 25)

seems quite similar to DecompressTarGzStream

Reply:
Remove this function。